src/neptune/new/handler.py

```diff
@@ -307,8 +307,34 @@ def append(
         wait: bool = False,
         **kwargs,
     ) -> None:
-        """
-        todo: NPT-12530
+        """Logs a series of values, such as a metric, by appending the provided value to the end of the series.
+
+        Available for following series field types:
+
+            * `FloatSeries` - series of float values
+            * `StringSeries` - series of strings
+            * `FileSeries` - series of files
+
+        When you log the first value, the type of the value determines what type of field is created.
```

Comment:
> When you log the first value, the type of the value determines what type of field is created.


This description is not accurate since `append` is build for single values only.

Reply:
What this means to say is: If there is no field "acc" from before, and you call

```
run["acc"].append(acc)
```

then if `acc` is a float, the field type will be FloatSeries.

Is this not accurate or is it just my phrasing that's confusing? Maybe this statement/explanation doesn't add much value?

Reply:
Ok, I get it. sorry.